cli.py

```diff
@@ -435,21 +435,21 @@ def resize_project_avatars():
                        # delete old avatar
                        obj = cont.get_object(old_avatar)
                        obj.delete()
-                       print "Done!"
+                       print("Done!")
                        cached_apps.get_app(a.short_name)
                    else:
-                       print "No Avatar found."
+                       print("No Avatar found.")
                 else:
-                   print "No avatar found."
+                   print("No avatar found.")
             except pyrax.exceptions.NoSuchObject:
-                print "Previous avatar not found, so not deleting it."
+                print("Previous avatar not found, so not deleting it.")
             except:
```

Comment:
- To catch any exception use the ``except Exception``, instead of 
  ``except BaseException`` or just ``except``. 
- Indicate the most specific exception type in ``except`` block. 

```python


     # Bad
     try:
         mapping[key]
     except Exception:
         ...

     # Better
     try:
         mapping[key]
     except KeyError:
         ...
```